.travis.yml

```diff
@@ -20,166 +17,144 @@ sudo: false
 
     # Don't wait for allowed failures
     fast_finish: true
 
     include:
+
         # Try MacOS X. Use a slightly old numpy version to help test against
         # all supported numpy versions.
-        - os: osx
-          stage: Cron tests
-          env: SETUP_CMD='test --remote-data=astropy'
-               CONDA_DEPENDENCIES="$CONDA_ALL_DEPENDENCIES clang"
-               PIP_DEPENDENCIES='jplephem asdf'
-               CCOMPILER=clang
-               EVENT_TYPE='cron'
+        # - os: osx
+        #   python: 3.6
+        #   # stage: Cron tests
+        #   env: TOXARGS='--remote-data=astropy'
+        #        CCOMPILER=clang
+        #        # CONDA_DEPENDENCIES="$CONDA_ALL_DEPENDENCIES clang"
+        #        # PIP_DEPENDENCIES='asdf'
+        #        # EVENT_TYPE='cron'
 
         # Try all python versions and Numpy versions. Since we can assume that
         # the Numpy developers have taken care of testing Numpy with different
         # versions of Python, we can vary Python and Numpy versions at the same
         # time.
-        # Run this test using native pytest
         - os: linux
-          env: PYTHON_VERSION=3.5 NUMPY_VERSION=1.13
-               INSTALL_CMD='python setup.py build_ext --inplace'
-               PIP_DEPENDENCIES='pytest-astropy'
-               TEST_CMD='pytest --open-files --doctest-rst'
-               PYTEST_VERSION=3.7
-          script:
-            - $INSTALL_CMD
-            - $TEST_CMD
+          python: 3.5
+          env: TOXENV=py35
+               TOXOPTS='--develop --force-dep \'pytest==3.7\''
+               TOXARGS='--cov=astropy --open-files --doctest-rst'
+               CFLAGS='--coverage -fno-inline-functions -O0'
 
         # Now try with all optional dependencies. We also include the --readonly
         # flag to make sure no files are being written to the temporary install
         # location during testing.
         - os: linux
-          env: PYTHON_VERSION=3.5 SETUP_CMD='test --remote-data=astropy --readonly'
-               CONDA_DEPENDENCIES=$CONDA_ALL_DEPENDENCIES
-               PIP_DEPENDENCIES="jplephem asdf"
-               LC_CTYPE=C.ascii LC_ALL=C
-               PYTEST_VERSION=3.7
+          python: 3.5
+          env: TOXENV=py35-alldeps
+               TOXOPTS='--force-dep \'pytest==3.7\''
+               TOXARGS='--cov=astropy --remote-data=astropy'
+               CFLAGS='--coverage -fno-inline-functions -O0'
+               # TOXARGS='--remote-data=astropy --readonly'
+               # LC_CTYPE=C.ascii LC_ALL=C
+               # PIP_DEPENDENCIES="asdf"
 
         - os: linux
           stage: Initial tests
-          env: PYTHON_VERSION=3.7 CONDA_DEPENDENCIES=$CONDA_ALL_DEPENDENCIES
-               PIP_DEPENDENCIES="asdf"
-               SETUP_CMD='test -a "--durations=50"'
+          python: 3.7
+          dist: xenial
+          sudo: true
+          env: TOXENV=py37-alldeps
+               TOXARGS='--durations=50'
+               # PIP_DEPENDENCIES="asdf"
```

Comment:
Why not just remove this line?

Reply:
Because I was not sure why it is specified here and not in pip-requirements. Most comments are to keep track of previous things that I needed to look at ;).